x/pairing/keeper/stakeModify.go

```diff
@@ -1,42 +1,42 @@
 package keeper
 
 import (
-	"fmt"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
-	"github.com/lavanet/lava/x/pairing/types"
 )
 
 func (k Keeper) CreditStakeEntry(ctx sdk.Context, chainID string, lookUpAddress sdk.AccAddress, creditAmount sdk.Coin) (bool, error) {
-	if creditAmount.Denom != epochstoragetypes.TokenDenom {
-		return false, fmt.Errorf("burn coin isn't right denom: %s", creditAmount.Denom)
-	}
-	// find the user in the stake list
+	// if creditAmount.Denom != epochstoragetypes.TokenDenom {
+	// 	return false, fmt.Errorf("burn coin isn't right denom: %s", creditAmount.Denom)
```

Comment:
we can't push it this way, can we remove the function and add a TODO in the caller?